### ISSUE TITLE
formula in table with table-layout:fixed;

### ISSUE THREAD

**Issue by sean1138** [mathjax/MathJax#1764]
mathjax formulas don't respect table cell padding when using table-layout:fixed; and width:100%;
example works to show issue in both firefox and chrome:
https://codepen.io/sean1138/pen/pwJwLB

after reading here:
https://css-tricks.com/fixing-tables-long-strings/
it seems to me that this might be expected behavior but not sure so still posting this.

Can be solved if setting a width (style="width:80px;" works in my example linked above) on the first cell of the first row is acceptable.  Setting a width on the first cell of the second row (the one with the formula) does not work because table-layout = fixed in the CSS I assume.

**Comment by pkra** [comment]
I think you are correct and this is expected behavor. `table-layout:fixed` fixes the layout by the first row, i.e., each column has the same width as the first cell.

You might want to enable automatic line-breaking in this case, http://docs.mathjax.org/en/latest/output.html#automatic-line-breaking,  or make the `.formula` wrapper scrollable.
